### ISSUE TITLE
Describe the logic of the software in README.md

### ISSUE THREAD

**Issue by ahojukka5** [VTT-ProperTune/OpenPFC#23]

**Comment by ahojukka5** [comment]
```mermaid
classDiagram
  App~Model~ --> Simulator
  Simulator --> Model
  Simulator --> Time
  Simulator --> FieldModifier
  Simulator --> ResultsWriter
  Model --> FFT
  Model --> RealField
  Model --> ComplexField
  FFT --> Decomposition
  Decomposition --> World
```

**Comment by ahojukka5** [comment]
Closed with commit 412cc71.
